.eslintrc.json

```diff
@@ -12,7 +12,14 @@
       "components": [ "Link" ],
       "specialLink": [ "hrefLeft", "hrefRight", "to" ],
       "aspects": [ "noHref", "invalidHref", "preferButton" ]
-    }]
+    }],
+    "jsx-a11y/label-has-for": [ 2, {
+      "components": [ "Label" ],
+      "required": {
+          "some": [ "nesting", "id" ]
+      },
+      "allowChildren": false
+  }]
   }
```

Comment:
![home](https://user-images.githubusercontent.com/25024356/32158629-b30fd7e0-bd6f-11e7-94c6-86af695f9193.PNG)
![postview](https://user-images.githubusercontent.com/25024356/32158630-b33ea25a-bd6f-11e7-8f00-2fd30626fe6c.PNG)
![studentlist](https://user-images.githubusercontent.com/25024356/32158631-b36fba3e-bd6f-11e7-9409-b1856fd4c7ec.PNG)


You can clone the internship portal repo in my account and test  it out.